docker-compose-perf.yml

```diff
@@ -469,6 +509,7 @@ services:
       echo -e 'Creating kafka topics'
       kafka-topics.sh --bootstrap-server kafka:29092 --create --if-not-exists --topic topic-transfer-prepare --replication-factor 1 --partitions $$KAFKA_GENERAL_PARTITIONS_NUM
       kafka-topics.sh --bootstrap-server kafka:29092 --create --if-not-exists --topic topic-transfer-position --replication-factor 1 --partitions $$KAFKA_POSITION_PARTITIONS_NUM
+      kafka-topics.sh --bootstrap-server kafka:29092 --create --if-not-exists --topic topic-transfer-position-batch --replication-factor 1 --partitions $$KAFKA_POSITION_BATCH_PARTITIONS_NUM
```

Comment:
Should this topic be added to `docker/kafka/scripts/provision.sh`?

Reply:
That script is being used in functional test stack `docker-compose.yaml` which is a recent refactor. I think we can add the new batch topic when we add batching to functional testing stack.